corporate/lib/stripe.py

```diff
@@ -1113,7 +1142,6 @@ def create_card_update_session(self) -> Dict[str, Any]:
             payment_method_types=["card"],
             success_url=f"{self.billing_session_url}/billing/event_status/?stripe_session_id={{CHECKOUT_SESSION_ID}}",
             billing_address_collection="required",
-            customer_update={"address": "auto"},
         )
```

Comment:
Why do we want to stop customer from updating address on the billing page update card session?

Reply:
So the address here is associated with the credit card that's being updated. This may or may not be the address that the customer wants on their invoices.

Since the billing portal for viewing and updating the name/address for the invoice (aka customer) is always visible and accessible on the billing page, it made more sense to me that the card update here not change any of the customer information, as they can update that information directly in the customer billing portal.

In the initial upgrade flow, we set both the name and address for the customer to be the same as the initial credit card data that's set/saved. This way, when/if the customer goes to the billing portal to review their billing information (either on the upgrade page or billing page), those fields are visible and are set to the data that they input for their card.

We could leave this to update the customer address to match the card address when the card is updated, but I think it might be confusing for folks who have explicitly set the address for invoices to not match the address on their credit card.

Reply:
Yeah, the current approach makes sense to me. E.g., you might have your personal address on the card, and the company's address on the invoice.